packages/sdk/src/observable/persistedObservable.ts

```diff
@@ -14,10 +14,7 @@ export type PersistedModel<T> =
     | { status: 'saving'; data: T }
     | { status: 'saved'; data: T }
 
-interface Storable {
-    tableName: string
-    load(): void
-}
+interface Storable {}
```

Comment:
do we still need it?

Reply:
probably not, but i don't know what to put in the decorator otherwise.